client/app/visualizations/chart/index.js

```diff
@@ -12,7 +12,10 @@ import editorTemplate from './chart-editor.html';
 const DEFAULT_OPTIONS = {
   globalSeriesType: 'column',
   sortX: true,
-  legend: { enabled: true },
+  legend: {
+    enabled: true,
```

Comment:
Why does the `legend` structure deviate from the Plotly API?

https://plot.ly/javascript/horizontal-legend/

E.g. `enabled` should probably be `showLegend`.

Reply:
Probably legacy reasons: the first implementation of the chart visualization used HighCharts, and the data structure was formed around its configuration...